CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ to set several of these new parameters.
 deprecation warnings.
 
 ### Fixed
+- Handling of antenna_names and antenna_numbers in `read_fhd` and `read_fhd_cal`.
```

Comment:
We just released a new version, so you need to rebase and move this up to be under the (new) unreleased section.